docs/api/non_geo_datasets.csv

```diff
@@ -1,34 +1,34 @@
 Dataset,Task,Source,# Samples,# Classes,Size (px),Resolution (m),Bands
 `ADVANCE`_,C,"Google Earth, Freesound","5,075",13,512x512,0.5,RGB
-`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,186x1,122",0.5,MSI
+`Benin Cashew Plantations`_,S,Airbus Pléiades,70,6,"1,122x1,186",10,MSI
 `BigEarthNet`_,C,Sentinel-1/2,"590,326",19--43,120x120,10,"SAR, MSI"
 `COWC`_,"C, R","CSUAV AFRL, ISPRS, LINZ, AGRC","388,435",2,256x256,0.15,RGB
 `Kenya Crop Type`_,S,Sentinel-2,"4,688",7,"3,035x2,016",10,MSI
-`Deep Globe Land Cover Challenge`_,S,DigitalGlobe +Vivid,803,7,"2,448x2,448",0.5,RGB
-`DFC2022`_,S,Aerial,,15,"2,000x2,000",0.5,RGB
+`Deep Globe Land Cover`_,S,DigitalGlobe +Vivid,803,7,"2,448x2,448",0.5,RGB
+`DFC2022`_,S,Aerial,"3,981",15,"2,000x2,000",0.5,RGB
 `ETCI2021 Flood Detection`_,S,Sentinel-1,"66,810",2,256x256,5--20,SAR
 `EuroSAT`_,C,Sentinel-2,"27,000",10,64x64,10,MSI
 `FAIR1M`_,OD,Gaofen/Google Earth,"15,000",37,"1,024x1,024",0.3--0.8,RGB
 `Forest Damage`_,OD,Drone imagery,"1,543",4,"1,500x1,500",,RGB
```

Comment:
Still don't know the resolution of this. It looks like Dan Morris (AI for Earth) is associated with this project and may know. There is also contact info on the dataset webpage: https://lila.science/datasets/forest-damages-larch-casebearer/

Reply:
Sent an email to the author

Reply:
FYI Dan is at google now